BraveRewardsUI/Common/DetailActionableRow.swift

```diff
@@ -0,0 +1,43 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+
+class DetailActionableRow: UIStackView {
```

Comment:
- Probably want the thing to act as a button and not just the arrow on the right side (similar to table view cell)
- Assuming you make this a `Button` subclass, can you add an animated `isHighlighted` state? (see other buttons, search isHighlighted) or make it highlight a BG around it? Something like this (apologizes for quick mockup):

<img width="572" alt="2" src="https://user-images.githubusercontent.com/529104/58709334-f245e000-8387-11e9-9457-81c13d18d7d2.png">

Let me know if it looks odd and just a alpha change would be better (like normal system buttons)

Reply:
Updated with alpha change.